api/src/main/java/openconsensus/metrics/data/Distribution.java

```diff
@@ -166,6 +166,18 @@ public static BucketOptions explicitOptions(List<Double> bucketBoundaries) {
      */
     public abstract Type getType();
 
+    /**
+     * Returns the bucket boundaries of this distribution.
+     *
+     * <p>Currently only {@link ExplicitOptions} implements this method.
+     *
+     * @return the bucket boundaries of this distribution.
+     * @since 0.1.0
+     */
+    public List<Double> getBucketBoundaries() {
+      throw new UnsupportedOperationException("Not implemented.");
```

Comment:
nit: message may say - "Only ExplicitOptions implements this method"

Reply:
Updated.